ghcide/src/Development/IDE/LSP/Outline.hs

```diff
@@ -66,11 +72,16 @@ moduleOutline ideState _ DocumentSymbolParams{ _textDocument = TextDocumentIdent
              in
                InR (InL allSymbols)
 
-
     Nothing -> pure $ InL []
 
-documentSymbolForDecl :: LHsDecl GhcPs -> Maybe DocumentSymbol
-documentSymbolForDecl (L (locA -> (RealSrcSpan l _)) (TyClD _ FamDecl { tcdFam = FamilyDecl { fdLName = L _ n, fdInfo, fdTyVars } }))
+getRefMap :: HieAstResult -> RefMap Type
+getRefMap HAR{refMap=refMap, hieKind=hieKind} =
+  case hieKind of
+    HieFromDisk _ -> mempty
```

Comment:
I have a question, why we use `mempty` for `HieFromDisk ` here

Reply:
This is mostly because I needed a `RefMap Type`, and I didn't know how to get there from `RefMap TypeIndex`. If this is important, I can keep hacking.

Reply:
Maybe you can find some inspiration for how to recover full types here?
https://github.com/haskell/haskell-language-server/blob/c11f32b62a199200951dbdcacce31516033ab031/ghcide/src/Development/IDE/Spans/AtPoint.hs#L298-L299